src/App.js

```diff
@@ -21,95 +21,41 @@ const messages = {
 addLocaleData([...en, ...es]);
 
 class App extends Component {
-  constructor() {
-    super();
+  constructor(props) {
+    super(props);
+
+    // TODO: Consider continue determining here default instance or moving all upside,
+    // forcing inject dependencies always
+    this.sessionManager =
+      (props.dependencies && props.dependencies.sessionManager) ||
+      new OnlineSessionManager(
+        new HttpDopplerMvcClient(axios, process.env.REACT_APP_API_URL),
+        60000,
+      );
+
     //TODO: this hardcoded data will depend by the app language
     const locale = navigator.language.toLowerCase().split(/[_-]+/)[0] || 'en';
```

Comment:
can we take from `headerData` maybe

Reply:
We will, but we need a default value before.